### ISSUE TITLE
Queries with errors are logged even if log-queries is disabled

### ISSUE THREAD

**Issue by captn3m0** [nextdns/nextdns#315]
I have occasional DNS query entries in my logs. I upgraded today to `1.8`, so these were made while running `1.7.1`. 

**Context**
 - Version [e.g. 1.5.2]: 1.7.1-1
 - Platform [e.g. macOS, ASUS-Merlin]: ArchLinux

Logs look like this:

```
Query 192.168.1.173 UDP A ssl.gstatic.com. (qry=33/res=12) cached HTTP/2.0: doh resolve: context deadline exceeded
Query 192.168.1.173 UDP A ogs.google.com. (qry=32/res=12) cached HTTP/2.0: doh resolve: context deadline exceeded
Query 192.168.1.173 UDP A zdnet4.cbsistatic.com. (qry=39/res=12) cached HTTP/2.0: doh resolve: context deadline exceeded
```

Or sometimes, if the upstream gives a 500:

```
Query 192.168.1.173 UDP A wss-primary.slack.com. (qry=39/res=12) cached HTTP/2.0: doh resolve: error code: 500
Query 192.168.1.173 UDP A wss-primary.slack.com. (qry=39/res=12) cached HTTP/2.0: doh resolve: error code: 500
```

Or on not found errors:

```
Query 192.168.1.222 UDP PTR lb._dns-sd._udp.0.1.168.192.in-addr.arpa. (qry=58/res=12) 0ms : not found
Query 192.168.1.222 UDP PTR 222.1.168.192.in-addr.arpa. (qry=44/res=12) 0ms : not found
Query 192.168.1.222 UDP PTR lb._dns-sd._udp.0.1.168.192.in-addr.arpa. (qry=58/res=12) 0ms : not found
Query 192.168.1.222 UDP PTR 222.1.168.192.in-addr.arpa. (qry=44/res=12) 0ms : not found
```

Found another one:

```
Query 192.168.1.222 UDP A time.asia.apple.com. (qry=37/res=12) cached HTTP/2.0: doh resolve: write tcp 192.168.1.111:53720->34.93.164.22:443: write: broken pipe
```

`log-queries` is set to false in the configuration. I have roughly ~300 entries per day for this in my logs. All of these include the client IP and a timestamp.

**Comment by rs** [comment]
This is expected so you can see errors but not be flooded with other queries. We will offer more log levels in the future.

**Comment by multiformeingegno** [comment]
BTW what does it mean when a query returns a “not found” error? I see a lot of these nextDNS logs on my routers. Almost a “not found” log per second. All UDP PTR xxx.in-addr.arpa

**Comment by rs** [comment]
It’s when local resolution (with discovery) get a nxdomain. This will go away.

**Comment by hazarjast** [comment]
This is listed as 'closed' with the last update being this behavior "will go away". Unfortunately it has not at least on the FreeBSD implementation. I am running the latest version of pfSense and my logs are still periodically flooded with PTR record "not found" entries for local LAN IP addresses. Would be nice to add a switch/option in "nextdns config" to mute these as pfSense default only shows last 50 entries to system log which is easily filled by these entries by nextdns pushing out more relevant info. Appreciate your feedback on this, TIA. 

Example log entries below for reference:
`

Time | Process | PID | Message
-- | -- | -- | --
Oct 12 07:56:55 | nextdns | 7318 | Query 192.168.1.131 UDP PTR lb._dns-sd._udp.254.254.254.10.in-addr.arpa. (qry=61/res=12) 0ms : not found
Oct 12 07:56:55 | nextdns | 7318 | Query 192.168.1.131 UDP PTR lb._dns-sd._udp.0.1.168.192.in-addr.arpa. (qry=58/res=12) 0ms : not found
Oct 12 08:04:33 | nextdns | 7318 | Query 192.168.1.91 UDP PTR 0.1.168.192.in-addr.arpa. (qry=42/res=12) 0ms : not found
||||...(many repeating entries of the same)...
Oct 12 08:56:09 | nextdns | 7318 | Query 192.168.1.92 UDP PTR 0.1.168.192.in-addr.arpa. (qry=42/res=12) 0ms : not found
Oct 12 09:00:00 | nextdns | 7318 | Received signal: terminated
Oct 12 09:00:00 | nextdns | 7318 | Stopping NextDNS 1.8.3/freebsd
Oct 12 09:00:00 | nextdns | 7318 | Restore router settings
Oct 12 09:00:00 | nextdns | 7318 | Deactivating
Oct 12 09:00:00 | nextdns | 85761 | Starting NextDNS 1.8.3/freebsd on :53
Oct 12 09:00:00 | nextdns | 85761 | Starting mDNS discovery
Oct 12 09:00:00 | nextdns | 85761 | Listening on UDP/:53
Oct 12 09:00:00 | nextdns | 85761 | Listening on TCP/:53
Oct 12 09:00:00 | nextdns | 7318 | NextDNS 1.8.3/freebsd stopped
Oct 12 09:00:00 | nextdns | 85761 | Connected 45.90.28.0:443 (con=51ms tls=160ms, TLS13)
Oct 12 09:00:01 | nextdns | 85761 | Connected 45.76.16.236:443 (con=49ms tls=54ms, TLS13)
Oct 12 09:00:01 | nextdns | 85761 | Switching endpoint: https://vultr-chi-1.edge.nextdns.io#45.76.16.236,2001:19f0:5c01:1ebc:5400:2ff:fece:28ed
Oct 12 09:00:05 | nextdns | 85761 | Setting up router
Oct 12 09:00:05 | nextdns | 85761 | Activating
Oct 12 09:04:39 | nextdns | 85761 | Query 192.168.1.91 UDP PTR 0.1.168.192.in-addr.arpa. (qry=42/res=12) 0ms : not found
Oct 12 09:04:44 | nextdns | 85761 | Query 192.168.1.91 UDP PTR 0.1.168.192.in-addr.arpa. (qry=42/res=12) 0ms : not found
Oct 12 09:06:06 | nextdns | 85761 | Query 192.168.1.92 UDP PTR 0.1.168.192.in-addr.arpa. (qry=42/res=12) 0ms : not found
Oct 12 09:06:11 | nextdns | 85761 | Query 192.168.1.92 UDP PTR 0.1.168.192.in-addr.arpa. (qry=42/res=12) 0ms : not found
||||...(many repeating entries of the same)...

`

**Comment by hazarjast** [comment]
For others' reference, this is resolved in version 1.8.6; just need to upgrade if you are on an older release.
